docs/faq/add_op_in_backend.md

```diff
@@ -318,7 +318,9 @@ parallel computation on both CPU and GPU. This allows most of
 the simple operators to share the same piece of code for CPU and GPU as
 parallelization approaches are often identical on both types of devices.
 The kernel function is defined as the following, where the function
-`Map` is executed by each thread for each input element. To explain a little
+`Map` is executed by each thread for each input element. The `out_data.Size()`
+specifies the number of threads that need to be spawned, which here corresponds
```

Comment:
This statement is not true. CPU and GPU kernels have different strategies of launching threads. Output size is not the number of threads in both cases.